scripts/dm/gladier.sh

```diff
@@ -12,7 +17,27 @@ METADATA_FILE_PATH=$4
 shift 4
 BOOST_CORR_ARGS=$@
 
+START_TIME=$(date +%s.%N)
 source $CONDA_PATH
+END_TIME=$(date +%s.%N)
+TIME_DIFF=$(awk "BEGIN {print $END_TIME - $START_TIME}")
+TIME_DIFF=$(printf "%.2f" "$TIME_DIFF")
+echo "Time sourcing conda script: $TIME_DIFF seconds"
+
+START_TIME=$(date +%s.%N)
 conda activate $CONDA_ENV
+END_TIME=$(date +%s.%N)
+TIME_DIFF=$(awk "BEGIN {print $END_TIME - $START_TIME}")
+TIME_DIFF=$(printf "%.2f" "$TIME_DIFF")
+echo "Time activating conda env: $TIME_DIFF seconds"
 
-python $DM_WORKFLOWS_DIR/scripts/xpcs_online_boost_client.py --experiment $EXPERIMENT --group $GROUP --hdf $METADATA_FILE_PATH $BOOST_CORR_ARGS
\ No newline at end of file
+START_TIME=$(date +%s.%N)
+python $DM_WORKFLOWS_DIR/scripts/xpcs_online_boost_client.py --experiment $EXPERIMENT --group $GROUP --hdf $METADATA_FILE_PATH $BOOST_CORR_ARGS
```

Comment:
The fact that `"$BOOST_CORR_ARGS"` is specified all as a separate entity makes me think we could support client usage in the `xpcs_online_boost_client.py` where we only track a very specific list of custom arguments, and treat all "unknown" arguments as boost corr arguments. This would potentially solve the problem where Miaoqi wants to add more custom arguments on-the-fly and run them, without necessarily needing to redeploy a new flow.